### PR TITLE
DOC Document changes to i18nTextCollector

### PR DESCRIPTION
**Pull request by GuySartorelli**
Documents the changes from https://github.com/silverstripe/silverstripe-framework/pull/10688 and https://github.com/silverstripe/silverstripe-framework/pull/10685 (don't merge this until those are both merged)

## Issue
- https://github.com/silverstripe/silverstripe-framework/issues/8491

### CODE REVIEW

File: en/02_Developer_Guides/13_i18n/index.md [hunk 1]
```diff
@@ -302,11 +302,28 @@ To collect all the text in code and template files we have just to visit: `https
...
+
+You can also run the text collector against multiple specific modules by separating the module names with a comma:
+`https://www.example.com/dev/tasks/i18nTextCollectorTask/?module=silverstripe%2Fcms,silverstripe%2Fframework`
+
+[info]
+The `%2F` in `silverstripe%2Fcms` is a `/` which has been encoded for use in a URL in a non-ambiguous way.
+[/info]
+
+The text collector also collects text for themes - if you want to run text collection on a specific theme, reference the theme
+with prefixed with `themes:`, e.g:
```
Comment (michalkleiner) [inline en/02_Developer_Guides/13_i18n/index.md]:
```suggestion
with prefix `themes:`, e.g:
```

Comment (ChloeSartorelli) [reply en/02_Developer_Guides/13_i18n/index.md]: Done

**Review by michalkleiner — APPROVED** [review]

**Comment by GuySartorelli** [comment]
This shouldn't have been merged yet - not all the actual code changes have been merged.
Not a big deal though i guess, they will be soon.

**Comment by michalkleiner** [comment]
Ah, my apologies @GuySartorelli, missed the note in the description. I guess keeping the PR in Draft would mean someone else would need to go and check if it can be merged if the author didn't change its status so it could prolong the process either way. Will check more carefully next time.

**Comment by GuySartorelli** [comment]
No worries, it happens.

Yeah it's hard sometimes to know if something should be draft or not. I probably would have made it draft if this was targeting a branch that would be live on merge, but `5.0` is the current live branch for docs so we're safe on that count.

**Pull request merged**